db/views/hmis_services_v01.sql

```diff
@@ -0,0 +1,35 @@
+SELECT
+	*
+FROM (
+	SELECT
+		"Services"."id" AS owner_id,
+		'Hmis::Hud::Service' AS owner_type,
+		"CustomServiceTypes"."id" AS custom_service_type_id,
+		"EnrollmentID",
+		"PersonalID",
+		"DateProvided",
+		"Services"."UserID",
+		"Services"."DateCreated" AS "DateCreated",
+		"Services"."DateUpdated" AS "DateUpdated",
+		"Services"."DateDeleted" AS "DateDeleted",
+		"Services"."data_source_id" AS "data_source_id"
+	FROM
+		"Services"
+		JOIN "CustomServiceTypes" ON "CustomServiceTypes"."hud_record_type" = "Services"."RecordType"
+			AND "CustomServiceTypes"."hud_type_provided" = "Services"."TypeProvided"
+			AND "CustomServiceTypes"."DateDeleted" IS NULL) hud_services
```

Comment:
@eanders what do you think?

Reply:
I think this looks good.  It makes me somewhat nervous to define this in SQL instead of generating the SQL using ActiveRecord.  You have `"CustomServiceTypes"."DateDeleted" IS NULL` but not the equivalent for the union or on the "Services" from.  That's probably ok, but the deleted records probably don't need to be exposed.(?)

Reply:
Yeah I don't think the deleted services need to be exposed, so I can add additional clauses for that.

I didn't even think about generating the SQL from AR. We'd need to add a relation between Services and CustomServicesTypes in order to do the join, which is fine. Do you think it is worth re writing this as an AR query and generating it from that? I'm not sure I really see the benefits to that, since we would still need to update the view SQL if any of the railsy logic changes.

Reply:
Let's go forward with this as is.  It seems safe and we're not 100% sure it'll do what we want in the long-run anyway.